### ISSUE TITLE
Alignment

### ISSUE THREAD

**Issue by hutchic** [OpenDevelopmentMethod/discussion#6]
If you're familiar with the alignment / autonomy talk from Spotify https://vimeo.com/85490944 the open development method I can imagine does great for autonomy. 

How do you maintain alignment with business objectives and goals though?

**Comment by ahmadnassri** [comment]
Yes, I am very familiar with that talk and a fan of the principle.

Agile Coaches tend to use that talk as a remedy for how "Agile" will work no matter what your team looks like.

The reality though is different, but I'm not going to discuss Agile in this thread. but i'll just note that nothing in that video (and little in the accompanying article that refer to Agile & Scrum as the process to use) in fact they actually emphasis that each squad has the choice of what method, tools, process to follow.

The alignment & autonomy approach lend itself quite well to the tenets of Discussion, Transparency & Democracy, and as oppose to Scrum teams with walled garden, Spotify's approach follows an open source model, where everybody can contribute to everything!
